frontend/src/containers/AddText.tsx

```diff
@@ -28,6 +29,7 @@ function AddText() {
   const [creatingWidget, setCreatingWidget] = useState(false);
   const [title, setTitle] = useState("");
   const [text, setText] = useState("");
+  const [showTitle, setShowTitle] = useState(true);
```

Comment:
Same here. No need to have `showTitle` in the state. Just get it from `getValues()` when you need it.